features/logging/deploy_logging_operators_via_console.feature

```diff
@@ -0,0 +1,77 @@
+    Then the step should succeed
+    Given "cluster-logging" packagemanifest's operator source name is stored in the :clo_opsrc clipboard
+    Given I switch to the first user
+    Given the first user is cluster-admin
+    Given I open admin console in a browser
+    # subscribe cluster-logging-operator
+    When I perform the :goto_operator_subscription_page web action with:
+      | package_name     | cluster-logging     |
+      | catalog_name     | <%= cb.clo_opsrc %> |
+      | target_namespace | openshift-logging   |
+    Then the step should succeed
+    Given I register clean-up steps:
+    """
+    Given logging service is removed successfully
+    Then the step should succeed
+    """
+    When I perform the :set_custom_channel_and_subscribe web action with:
+      | update_channel    | <%= cb.logging_channel %> |
+      | install_mode      | OwnNamespace              |
+      | approval_strategy | Automatic                 |
+    Then the step should succeed
+    Given cluster logging operator is ready
+
+  # @auther qitang@redhat.com
+  # @case_id OCP-24292
+  @admin
+  @destructive
+  Scenario: Deploy elasticsearch-operator via Web Console
+    # clear the logging related resources
+    Given logging service is removed successfully
+    Given I switch to cluster admin pseudo user
+    Given logging channel name is stored in the :logging_channel clipboard
+    Given "elasticsearch-operator" packagemanifest's operator source name is stored in the :eo_opsrc clipboard
+    Given I switch to the first user
+    Given the first user is cluster-admin
+    Given I open admin console in a browser
+    When I perform the :goto_operator_subscription_page web action with:
+      | package_name     | elasticsearch-operator |
+      | catalog_name     | <%= cb.eo_opsrc %>     |
+      | target_namespace | openshift-operators    |
+    Then the step should succeed
+    When I perform the :set_custom_channel_and_subscribe web action with:
+      | update_channel    | <%= cb.logging_channel %> |
+      | install_mode      | AllNamespace              |
+      | approval_strategy | Automatic                 |
+    Then the step should succeed
+
+    Given I use the "openshift-operators" project
+    Given I wait for the "elasticsearch-operator" subscriptions to appear
+    And evaluation of `subscription("elasticsearch-operator").current_csv` is stored in the :eo_csv clipboard
+    Given I register clean-up steps:
+    """
+    Given I use the "openshift-operators" project
+    When I run the :delete client command with:
+      | object_type       | sub                    |
+      | object_name_or_id | elasticsearch-operator |
+      | namespace         | openshift-operators    |
+    Then the step should succeed
+    When I run the :delete client command with:
+      | object_type       | csv                 |
+      | object_name_or_id | <%= cb.eo_csv %>    |
+      | namespace         | openshift-operators |
+    """
```

Comment:
if you want to delete something, better use step:
```
/^(I|admin) ensures? #{QUOTED} (\w+) is deleted(?: from the#{OPT_QUOTED} project)?( after scenario)?$/
```

Also clean-up steps need to be registered imediately before or after resource to be removed was created.